src/components/agencies/view-agency/view-agency.component.js

```diff
@@ -128,44 +160,112 @@ class ViewAgency extends Component {
           <div className="flex-row">
             {1 === activeTab && (
               <div className="full-view white-bg box-shadow agency-tab-content">
-                <table className="custom-table">
-                  <thead>
-                    <tr className="border-bottom">
-                      <td>
-                        <div className="flex-row align-center justify-between agency-info-box">
-                          <div className="table-name">
-                            {`${t("TABLE.OFFICERS")} (${
-                              agencyAdditionalInfo.officers.length
-                            })`}
-                          </div>
-                          <button className="white-btn">
-                            {t("BUTTONS.CREATE_REPORT")}
-                          </button>
-                        </div>
-                      </td>
-                    </tr>
-                  </thead>
-                  <tbody>
-                    {agencyAdditionalInfo.officers
-                      ? agencyAdditionalInfo.officers.map((officer, ind) => (
-                          <tr className="table-row" key={ind}>
-                            <td>
-                              <div className="flex-row align-center agency-info-box">
-                                <div className="officer-container-img">
-                                  <img
-                                    className="icon"
-                                    src={require("../../../assets/crew-icon.png")}
-                                    alt="no logo"
+                <div className="flex-row align-center justify-between agency-info-box">
+
+                  <div className="table-name margin-left">
+                    {`${agencyAdditionalInfo.officers.length} ${t("TABLE.OFFICERS")}`}
+                  </div>
+
+                  <div className="flex-row align-center justify-between">
+                    <div className={`flex-row officer-search-panel search ${isFocused ? "focused" : ""}`}>
+                      <div className="flex-row align-center search-icon">
+                        <SearchIcon htmlColor={isFocused ? `#0a4074` : "#979797"} />
+                      </div>
+                      <input
+                        className="search-field officer-search-panel-input"
+                        type="search"
+                        placeholder={`${t("SEARCH.FILTER_SEARCH")} O-FISH`}
+                        onChange={this.setSearch}
+                        onFocus={() => this.setState({ isFocused: true })}
+                      ></input>
+                    </div>
+
```

Comment:
Is search supposed to work? You added the UI components, which is awesome. I'm happy to make another ticket for search to work, but if you intended it to work it does not. Just want to make sure we're on the same page!

Reply:
I think for current issue search functionality doesn't have to work & also separate ticket for search, issue #213 is already present, which is dependent on issue #212